app.js

```diff
@@ -0,0 +1,5 @@
+import config from './config/config';
+import * as models from './models';
+
+const user = new models.User();
```

Comment:
// app.js
```
import { User, Product } from './models';

// code...
```

Using ` new models.User()` unreadable as for me.
Try codestyle above, please.

Reply:
Done.